pkg/query-service/app/explorer/db.go

```diff
@@ -0,0 +1,157 @@
+package explorer
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
+	v3 "go.signoz.io/signoz/pkg/query-service/model/v3"
+)
+
+var db *sqlx.DB
+
+type ExplorerQuery struct {
+	ID         int64     `json:"id" db:"id"`
+	UUID       string    `json:"uuid" db:"uuid"`
+	CreatedAt  time.Time `json:"created_at" db:"created_at"`
+	UpdatedAt  time.Time `json:"updated_at" db:"updated_at"`
+	SourcePage string    `json:"source_page" db:"source_page"`
+	// 0 - false, 1 - true
+	IsView    int8   `json:"is_view" db:"is_view"`
+	Data      string `json:"data" db:"data"`
+	ExtraData string `json:"extra_data" db:"extra_data"`
+}
+
+// InitWithDSN sets up setting up the connection pool global variable.
+func InitWithDSN(dataSourceName string) (*sqlx.DB, error) {
+	var err error
+
+	db, err = sqlx.Open("sqlite3", dataSourceName)
+	if err != nil {
+		return nil, err
+	}
+
+	tableSchema := `CREATE TABLE IF NOT EXISTS explorer_queries (
+		id INTEGER PRIMARY KEY AUTOINCREMENT,
```

Comment:
Why not use `uuid` as primary key?

Reply:
We could do that as well. I don't remember why I did this. Will update.

Reply:
uuid is primary key now